src/app/components/shared/page/Page.tsx

```diff
@@ -0,0 +1,23 @@
+import { FullHeightSpinner } from '@app/components/shared/spinner/FullHeightSpinner';
+import { ReactNode, Suspense } from 'react';
+
+interface Props {
```

Comment:
**suggestion** In React there is a type available: `PropsWIthChildren` why did you decide to declare the children type yourself?